pkg/grizzly/formatting.go

```diff
@@ -11,27 +14,26 @@ func Format(registry Registry, resourcePath string, resource *Resource, format s
 	var extension string
 	var err error
 
-	spec := resource
+	spec := resource.Body
 	if onlySpec {
-		s := Resource(resource.Spec())
-		spec = &s
+		spec = resource.Spec()
 	}
 
-	switch format {
-	case "yaml":
-		extension = "yaml"
-		content, err = spec.YAML()
-	case "json":
+	if format == "json" {
 		extension = "json"
-		content, err = spec.JSON()
-	default:
+		j, err := json.MarshalIndent(spec, "", "  ")
+		if err != nil {
+			return nil, "", "", err
+		}
+		content = string(j)
```

Comment:
We convert this from []byte to string to []byte. Looks like this could be simplified :)